webcompat/static/css/src/grid.css

```diff
@@ -1,5 +1,5 @@
 .grid {
-  margin: 9px;
+  margin: 9px 0;
```

Comment:
Should we use the var here too?

Reply:
This is probably not related / a side effect. It was the initial overall margin for all `.grid` elements and therefore produced the left and right margin.

Reply:
Just to make sure, we don't misunderstand each other, this was referring to the `9px` and if we should change this to `--grid-outermargin-s`. But this is 100% not blocking and we are free to merge!